keyboards/kyria/keymaps/gotham/rules.mk

```diff
@@ -0,0 +1,12 @@
+CONSOLE_ENABLE = no        # Console for debug
+ENCODER_ENABLE = yes       # ENables the use of one or more encoders
+RGBLIGHT_ENABLE = yes      # Enable keyboard RGB underglow
+MOUSEKEY_ENABLE = no       # Mouse keys
+OLED_DRIVER_ENABLE = yes   # Enables the use of OLED displays
+
+JOYSTICK_ENABLE = yes
```

Comment:
Should be renamed along with define to avoid conflicts with #4226. thumbstick maybe?

Reply:
Thumbstick sounds good. I'll get right on it.